### ISSUE TITLE
Crowdsource point edited to Forward Progress Stopped clears FM point

### ISSUE THREAD

**Issue by KeithJGw** [pjdohertygis/FireMappers#41]
Dutch fire edited from active to Forward Progress Stopped clears point in All PIMs; 2.0, 3.0, & 3.0 Media.
Additional: Edit was in FM 2.0 Admin: https://experience.arcgis.com/experience/d2933dba9fa64df1bddc016825b54e07
Pt removed in all PIMS FM 2.0 https://napsg.maps.arcgis.com/apps/webappviewer/index.html?id=6dc469279760492d802c7ba6db45ff0e
3.0 https://experience.arcgis.com/experience/caebb09ac0fb44799609b440fa320280
and 3.0 Media: https://napsg.maps.arcgis.com/apps/instant/media/index.html?appid=a483112967f84db096e1c954f8249ad6

**Comment by pjdohertygis** [comment]
@KeithJGw can you add links to the apps where there are issues just so we are explicit? Is this in 2.0 or 3.0 or both?

**Comment by pjdohertygis** [comment]
I think this is related to the custom symbology @afackler setup which appears to be busted. I see four points in the feature layer view that pass through the filter, but only one point on the map. 

I see this in both 2.0 https://napsg.maps.arcgis.com/apps/mapviewer/index.html?webmap=640ca58af80142baa2b5155854fbbb2f

and 3.0 https://napsg.maps.arcgis.com/apps/mapviewer/index.html?webmap=58f752056455490f8040a24cb7fdc536

![image](https://user-images.githubusercontent.com/4256979/190243337-a3f73230-0dc9-4d50-841e-55f660d68b14.png)

![image](https://user-images.githubusercontent.com/4256979/190243549-6412da01-c90c-420a-8e0b-596eaad8a687.png)

When I reset the symbology, the issue goes away.
![image](https://user-images.githubusercontent.com/4256979/190243682-e8e83800-ec2d-4633-b4d3-e7da45cd0a78.png)

I'll check with Adam and see if he has any idea.

**Comment by pjdohertygis** [comment]
@KeithJGw - I've removed the custom code from the feature layer view and reset the symbology thanks to @afackler setting up the FireMappers icons as a style in the NAPSG org. 

![image](https://user-images.githubusercontent.com/4256979/190256642-407f89f2-e7d3-4c7f-a30f-147afc97e8eb.png)


This should give us some flexibility to play around with feature layer view filters again. The problem was we had filters set up in multiple places. 

1) Feature Layer View definition
2) The Web Map 
3) Custom Symbology
4) WAB has a group filter widget which I don't remember exactly why we made it

With all this complexity I think we painted ourselves into a corner.  Can you test... 

Web Map 3.0 https://napsg.maps.arcgis.com/apps/mapviewer/index.html?webmap=58f752056455490f8040a24cb7fdc536
Web Map 2.0 https://napsg.maps.arcgis.com/apps/mapviewer/index.html?webmap=640ca58af80142baa2b5155854fbbb2f
Web App 3.0 (ExB) https://experience.arcgis.com/experience/caebb09ac0fb44799609b440fa320280
Web App 2.0 (WAB) https://www.arcgis.com/apps/webappviewer/index.html?id=6dc469279760492d802c7ba6db45ff0e

Note for 2.0 I had to downgrade the symbology because it cannot use the symbols (either because Adam uploaded as svg or some other issue) 
![image](https://user-images.githubusercontent.com/4256979/190259282-9816f821-26ab-4543-9ace-6abc9e5354dc.png)

**Comment by pjdohertygis** [comment]
@KeithJGw I created a new "fall-off" rule to avoid using dates which is what is causing the issue. 

This means you get to be the guy who decides what is relevant / not relevant for the time being : ) 

![image](https://user-images.githubusercontent.com/4256979/190276782-a5dabcc1-6a9a-4480-bf03-93006d052d73.png)
![image](https://user-images.githubusercontent.com/4256979/190276937-a8e2bfdc-9cb6-4463-9405-d645f9c4bf13.png)

Basically just say "yes show on map" for that question, and then no when you think it should go away.
